templates/_macros/vf_cta-section.jinja

```diff
@@ -0,0 +1,105 @@
+
+# All Slots
+  # title: Optional title slot. Only applicable to "cta-default-100" and "cta-default-25-75" layouts. When specified, the "title_text" parameter will be ignored.
+  # description: Paragraph-style (one or more) content below the title. This slot is required for "cta-block-100" and "cta-block-25-75" layouts.
+  # cta: Call-to-action block with action links/buttons. This slot is required for "cta-block-100" and "cta-block-25-75" layouts.
+
+# Variants:
+  # cta-default-100: Full-width CTA with title and link text
+    # Params:
+        # title_text: H2 title text - optional
+        # link_text: H2 link text - required
+        # link_href: href for the h2 link - required
+        # layout: the variant - cta-default-100
+        # has_chevron: Whether the link text should have a chevron icon - Optional, default is True
+
+    # Slots:
+        # title: When specified, the "title_text" parameter will be ignored - Optional
+
+  # cta-default-25-75: 25-75 percent width CTA with title and link text
+    # Params:
+        # title_text: H2 title text - optional
+        # link_text: H2 link text - required
+        # link_href: href for the h2 link - required
+        # layout: the variant - cta-default-25-75
+        # has_chevron: Whether the link text should have a chevron icon - Optional, default is True
+
+      # Slots:
+        # title: When specified, the "title_text" parameter will be ignored - Optional
+
+  # cta-block-100: Full width CTA with title, description and cta block with action links
+    # Params:
+        # title_text: H2 title text - required
+        # layout: the variant - cta-block-100
+
+    # Slots:
+        # description: Paragraph-style (one or more) content below the title - Optional
+        # cta: Call-to-action block (required)
+
+  # cta-block-25-75: 25-75 percent width CTA with title, description and cta block with action links
+    # Params:
+        # title_text: H2 title text - required
+        # layout: the variant - cta-block-25-75
+
+    # Slots:
+        # description: Paragraph-style (one or more) content below the title - Optional
+        # cta: Call-to-action block (required)
+
+{% macro vf_cta_section(title_text, link_text, link_href, has_chevron=True, layout='cta-default-100', caller=None) %}
+  {% set title_content = caller('title') %}
+  {% set has_title_slot = title_content|trim|length > 0 %}
+  {% set description_content = caller('description') %}
+  {% set has_description = description_content|trim|length > 0 %}
+  {% set cta_content = caller('cta') %}
+  {% set has_cta = cta_content|trim|length > 0 %}
+  {% macro _description_block() -%}
+    {% if has_description %}{{ description_content }}{% endif %}
+  {% endmacro %}
+  {% macro _cta_link() %}
+    {% if link_text and link_href %}
+      <a href="{{ link_href }}">{{ link_text }}
+        {% if has_chevron %}&#32;&rsaquo;{% endif %}
+      </a>
+    {% endif %}
```

Comment:
This limits the attributes that can be passed to the link. Might there ever be a case for additional attributes, like `target`, `download`, `rel`, `hreflang`, `aria-label`, etc? If so, I'd suggest allowing the user to pass in a dict of link attributes, in addition to link_text, rather than restricting this.